### PR TITLE
Many brainmed changes.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
This PR addresses three main points:
- Lethality;
- liver necrosis;
- blood donations.

To address lethality, lasers now deal more bloodloss.

To address liver necrosis, I removed the existing liver code and ported some chunks of Bay's liver and retrofitted it a bit.

To address blood donations, the hardest issue, IVs now ping when the patient is below 90% blood volume.

To address shrapnel, it now does external damage. Yeowch.

I also fixed a few pain messages here and there.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: html/changelogs/mattatlas-braincase.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "IVs now ping when the attached person is at less than 90 blood volume."
```
Comment (Geevies) [inline html/changelogs/mattatlas-braincase.yml]:
```suggestion
  - rscadd: "IVs now ping when the attached person is at less than 90% blood volume."
```

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -1120,7 +1120,8 @@
 		custom_pain("[pick("The pain is excruciating", "Please, just end the pain", "Your whole body is going numb")]!", 40, nohalloss = TRUE)
 
 	if (shock_stage >= 60)
-		if(shock_stage == 60) emote("me",1,"'s body becomes limp.")
+		if(shock_stage == 60) 
+			emote("me",1,"'s body becomes limp.")
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/life.dm]:
```suggestion
			visible_message("[src]'s body becomes limp.", "Your body becomes limp.")
```
Pretty sure this caused issues like `name 's body becomes limp`, iirc.

File: code/game/machinery/iv_drip.dm [hunk 3]
```diff
@@ -111,12 +112,19 @@
...
+					visible_message("\The [src] pings.")
 				return
 
+			if(T.get_blood_volume() < 90 && !blood_message_sent)
+				visible_message("\icon[src] \The <b>[src]</b> flashes a warning light!")
+				playsound(src, 'sound/machines/buzz-two.ogg', 50)
+				blood_message_sent = TRUE
+
 			var/datum/reagent/B = T.take_blood(beaker,amount)
+			B.volume += amount * 0.8
```
Comment (MarinaGryphon) [inline code/game/machinery/iv_drip.dm]: I'm not so sure about this. Just... make blood bags smaller. We already have enough infinite chem duplication glitches as it is, we don't need to intentionally add one.

**Pull request merged**